ui/assets/css/_header.scss

```diff
@@ -0,0 +1,19 @@
+.usa-disclaimer {
+  background-color: $color-gray-lightest;
+  color: $color-gray-dark;
+  font-size: 12px;
```

Comment:
@cmc333333 for this one in particular I don't know if we will use it anywhere else but I think we should definitely do that for some of the UI text. 

@theresaanna I'll make a separate task to establish a baseline and figure on what kind of flexible unit we want to go with